webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7361,20 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.checkBotDetail = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/application/" + MCK_APP_ID + "/bot/" + CURRENT_GROUP_DATA.conversationAssignee,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                                CURRENT_GROUP_DATA.CHAR_CHECK = data.data[0] ? ((data.data[0].aiPlatform == 'dialogflow' && !((data.data[0]).autoHumanHandoff)) ? true : false) : false ;
```

Comment:
@hardikparashar You can use logical operators as they will return boolean values itself.
Check if this works : 
**CURRENT_GROUP_DATA.CHAR_CHECK = data.data[0] && (data.data[0].aiPlatform == 'dialogflow' && !((data.data[0]).autoHumanHandoff));**

+ add all bot platforms in an object (km-utils.js).